File: ProjectEuler_CSharp/EulerProblem1.cs
using System.Linq;

namespace ProjectEuler_CSharp
{
    public static class EulerProblem1
    {
        public static long Calculate(int maxValue)
            => Enumerable.Range(0, maxValue).Where(n => n % 3 == 0 || n % 5 == 0).Sum();
    }
}

File: ProjectEuler_FSharp/EulerProblem1.fs
module ProjectEuler_FSharp.EulerProblem1

let Calculate(maxValue : int32) =
    seq {1..maxValue - 1}
    |> Seq.filter (fun n -> n % 3 = 0 || n % 5 = 0)
    |> Seq.sum